combined_robot_hw_tests/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(combined_robot_hw_tests)
```

Comment:
Do the tests really need their own package? Can't they just be inside the main package?